ArduCopter/mode_auto.cpp

```diff
@@ -286,7 +310,7 @@ void ModeAuto::circle_movetoedge_start(const Location &circle_center, float radi
         circle_edge.set_alt_cm(circle_center.alt, circle_center.get_alt_frame());
 
         // initialise wpnav to move to edge of circle
-        if (!wp_nav->set_wp_destination(circle_edge)) {
+        if (!wp_nav->set_wp_destination_loc(circle_edge)) {
```

Comment:
Leonard added the "_loc" because he found it difficult to follow the flow through the various similarly named methods.  My view is if it helps a key developer and doesn't impact others too much then it is OK.

I tried to extract the change into a separate PR but it's too wound up in everything else.

Reply:
I am happy for this change to be reverted out. I put it in there because I was trying to work out the state machine flow. That purpose has passed.